src/main/java/com/arcbees/analytics/server/AnalyticsProxyServlet.java

```diff
@@ -30,18 +30,19 @@
 @Singleton
 public class AnalyticsProxyServlet extends HttpServlet {
 
```

Comment:
usually,we don't botter about the empty line when the definition of the class is on more than one line